### ISSUE TITLE
绿色版mac和windows都报错

### ISSUE THREAD

**Issue by suifengqjn** [xinntao/Real-ESRGAN#466]
当我指定模型
./gan -i ./input.jpg -o input_out.jpg -m ./models/realesrgan-x4plus-anime.bin
[1]    99330 segmentation fault  ./gan -i ./input.jpg -o input_out.jpg -m ./models/realesrgan-x4plus-anime.bin

不指定模型
./gan -i ./input.jpg -o input_out.jpg                                        
[0 AMD Radeon Pro 5300M]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[0 AMD Radeon Pro 5300M]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 AMD Radeon Pro 5300M]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 AMD Radeon Pro 5300M]  subgroup=64  basic=1  vote=1  ballot=1  shuffle=1
[1 Intel(R) UHD Graphics 630]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[1 Intel(R) UHD Graphics 630]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[1 Intel(R) UHD Graphics 630]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[1 Intel(R) UHD Graphics 630]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1
gan(1346,0x1176e6e00) malloc: Incorrect checksum for freed object 0x7fe8f206be00: probably modified after being freed.
Corrupt value: 0x2020226c75762f
gan(1346,0x1176e6e00) malloc: *** set a breakpoint in malloc_error_break to debug
[1]    1346 abort      ./gan -i ./input.jpg -o input_out.jpg

以上是在macbook pro 执行
![image](https://user-images.githubusercontent.com/11765306/195477227-85736e7d-1e63-4fbe-a917-40cd85f4ecc0.png)

**Comment by Franklalalala** [comment]
metoo, waiting for response, though windows version has no problem.

**Comment by Basicconstruction** [comment]
yes, i meet same result in ubuntu, it sames that they forgot to include the .so file
